plugins/chat/assets/javascripts/discourse/services/chat-api.js

```diff
@@ -373,6 +373,18 @@ export default class ChatApi extends Service {
     );
   }
 
+  /**
+   * Update thread title prompt of current user for a thread.
+   * @param {number} channelId - The ID of the channel.
+   * @param {number} threadId - The ID of the thread.
+   * @returns {Promise}
+   */
+  updateCurrentUserThreadTitlePrompt(channelId, threadId) {
+    return this.#putRequest(
```

Comment:
Should probably be a post given, you can only set to true

Reply:
Previously we were passing a value here, but I removed that so changing this to post is probably fine